crates/fuel-core/src/graphql_api/worker_service.rs

```diff
@@ -356,6 +361,32 @@ where
             )
             .into());
         }
+
+        for receipt in result.receipts() {
+            match receipt {
+                Receipt::Mint {
+                    sub_id,
+                    contract_id,
+                    ..
+                }
+                | Receipt::Burn {
+                    sub_id,
+                    contract_id,
+                    ..
+                } => {
+                    let asset_id = AssetId::from(**contract_id);
+                    let current_count = db
```

Comment:
You need to increase/decrease based on the minted/burned amount=)

Reply:
Good catch 😄 ab09b8df08